### PR TITLE
Update runtime to 5.15

### LINKED ISSUE #2: update KDE runtime to 5.15

### PR DESCRIPTION
**Pull request by hfiguiere**
- also make the build reproducible by using tag+commit

Close #2

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build

**Comment by hfiguiere** [comment]
I had run out of disk space, to I didn't test the build completely. Will fix this.

**Comment by hfiguiere** [comment]
Updating PyQt was deemed necessary to build.

File: flathub.json [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
-  "only-arches": ["aarch64", "x86_64"]
```
Comment (aleixpol) [inline flathub.json:2]: Are you sure this is not necessary anymore?

File: qgis.json [hunk 2]
```diff
@@ -5,15 +5,16 @@
         {
             "type" : "git",
             "url" : "https://github.com/kadas-albireo/QGIS.git",
-            "branch" : "kadas_modifications"
+            /* "branch" : "kadas_modifications", */
```
Comment (aleixpol) [inline qgis.json]: I've seen comments break tooling once and again. You can make the key "//branch" and it should work.

**Review by aleixpol — COMMENTED** [review]
What's the final status? Has this been tested? @manisandro.

File: flathub.json (hunk 1 above)
Comment (hfiguiere) [reply flathub.json:2]: these are the only two arches available on this runtime.

File: qgis.json (hunk 2 above)
Comment (hfiguiere) [reply qgis.json]: we can just remove it, instead. I have never had any problem, but flathub-extra-data-checker stripping them.

Comment (hfiguiere) [reply qgis.json]: I used an alternative, JSON compatible way to do it. The important point here is to indicate the branch for the maintainer, we can't use branch + commit nor a bare branch.

**Comment by hfiguiere** [comment]
any chance to get this merged?

**Comment by aleixpol** [comment]
I guess we can merge, I was hoping for a comment from @manisandro.

**Pull request merged**